app/email.go

```diff
@@ -321,14 +321,23 @@ func (a *App) SendInviteEmails(team *model.Team, senderName string, senderUserId
 		return
 	}
 
+	sender, appErr := a.GetUser(senderUserId)
+	if appErr != nil {
+		a.Log().Error("Email invite not sent, unable to find the sender user.",
+			mlog.String("user_id", senderUserId),
+			mlog.String("team_id", team.Id),
+			mlog.Err(appErr))
+		return
+	}
+
 	for _, invite := range invites {
 		if len(invite) > 0 {
 			subject := utils.T("api.templates.invite_subject",
 				map[string]interface{}{"SenderName": senderName,
 					"TeamDisplayName": team.DisplayName,
 					"SiteName":        a.ClientConfig()["SiteName"]})
 
-			bodyPage := a.newEmailTemplate("invite_body", model.DEFAULT_LOCALE)
+			bodyPage := a.newEmailTemplate("invite_body", sender.Locale)
```

Comment:
We may not need to get the sender user. I believe that if you pass here empty locale it will default to `utils.T` that uses the server set language by default. See here: https://github.com/mattermost/mattermost-server/blob/71925ea224666ce2f869f2607cde71341c4fa6d5/app/email.go#L469-L474

Reply:
Fair enough. That's much better. Thanks.